sdk/eventhub/event-hubs/samples/useWithIotHub.ts

```diff
@@ -12,7 +12,7 @@ import { EventHubConsumerClient } from "@azure/event-hubs";
 const connectionString = "";
 
 async function main(): Promise<void> {
-  const client = new EventHubConsumerClient(connectionString);
+  const client = new EventHubConsumerClient(EventHubConsumerClient.defaultConsumerGroup, connectionString);
```

Comment:
Please log an issue to consider not using the `EventHubConsumerClient.defaultConsumerGroup` in our samples or adding the "name" back just for this constant.

Now that the word "name" has gone out of the name of this constant, I worry that it will mislead users to assume that this will always point to the default consumer group which might not be the case if the default consumer group's name has been changed.

Reply:
I added it back for the name of the constant and scrubbed it from our samples. :|